### PR TITLE
Extract default config into their own files

### PR DESCRIPTION
**Pull request by froi**
Much of the work that is being done is copying config files for every municipality and playing with the values until the extract is just right. This can be very confusing for some people that are new to the project.

This PR attempts to extract the though of what configs are needed by putting them all into their own file and exporting a generator type function that will return the config for a specified config.

### CODE REVIEW

File: papeleta-pdf-extractor/src/config/2020/base-config.ts [hunk 1]
```diff
@@ -0,0 +1,223 @@
+export interface BaseRow {
```
Comment (Jpadilla1) [inline papeleta-pdf-extractor/src/config/2020/base-config.ts:1]: Would there be more cases where we want to extend BaseRow?

File: papeleta-pdf-extractor/src/config/2020/base-config.ts [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+  N_LOGO?: number;
+  LOGO_TOP?: number;
+  LOGO_HEIGHT?: number;
+  LOGO_LEFT_DELTA?: number;
+  LOGO_RIGHT_DELTA?: number;
+}
+interface DefaultValues {
+  [index: string]: BaseRow | Row;
+}
+const defaultRowValues: DefaultValues = {
```
Comment (Jpadilla1) [inline papeleta-pdf-extractor/src/config/2020/base-config.ts:17]: The default values are for a legislative ballot and not general purpose. We might want to have separate configs or base classes (Legislative, Municipal, etc) instead.

How would a config look like using this method?

**Comment by Nosferican** [comment]
I used the same configs as for the legislative one (see [L19-L37](https://github.com/Nosferican/CandidatosEleccionesGeneralesPR2020/blob/ec3e0c1a1a606e26b4850bd9dcd1ff388a63cbd2/src/municipal.jl#L19-L37)).

File: papeleta-pdf-extractor/src/config/2020/base-config.ts (hunk 2 above)
Comment (froi) [reply papeleta-pdf-extractor/src/config/2020/base-config.ts:17]: @Jpadilla1 This is just an initial stab at it. 

If this approach would be something that we can use I'd make base configs for municipal and legislative ballots. Making a config module that exposes a single API for configs will be useful IMO.

If we want to continue with this I'll structure it better and make this useable.

File: papeleta-pdf-extractor/src/config/2020/base-config.ts (hunk 1 above)
Comment (froi) [reply papeleta-pdf-extractor/src/config/2020/base-config.ts:1]: Not that I can see as of yet. If you think we should continue with this effort then it's a matter of taking a look at the municipal configs to see if this `BaseRow` would be extended later.

**Comment by froi** [comment]
> I used the same configs as for the legislative one (see [L19-L37](https://github.com/Nosferican/CandidatosEleccionesGeneralesPR2020/blob/ec3e0c1a1a606e26b4850bd9dcd1ff388a63cbd2/src/municipal.jl#L19-L37)).

Seeing this makes me thing we are on a good path to help automate this to some extent. @Jpadilla1 thoughts?

**Comment by froi** [comment]
Closing this out as not need/wanted at this time for this repo. We'll reuse this once we finish work and get a breather to extract the parser from this repo.

**Pull request closed without merging**